src/v/cluster/feature_table.h

```diff
@@ -107,6 +108,12 @@ constexpr static std::array feature_schema{
     feature::serde_raft_0,
     feature_spec::available_policy::always,
     feature_spec::prepare_policy::always},
+  feature_spec{
+    cluster_version{5},
```

Comment:
please update `static constexpr cluster_version latest_version` in `feature_table.cc`